test/test_gen.py

```diff
@@ -9,6 +9,8 @@
 
 from test.helpers import get_inputdata_file
 
+bc_s2_path = '/home/alicja/Desktop/bc/testdata/bc_sentinel_2/coast/'
```

Comment:
Is it possible to save a small subset of this file and add it to the repo and use it in the test? Not a blocker.

Reply:
Currently not possible - I talked to Carole and she preferred not publishing them. Maybe we can find some suitable demo datasets in the future, which are ok to be published. :)